src/OpenTelemetry/CHANGELOG.md

```diff
@@ -56,6 +56,9 @@ Released 2021-Mar-19
 * Added `SetResourceBuilder` support to `OpenTelemetryLoggerOptions`.
   ([#1913](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1913))
 
+* `ResourceBuilderExtensions` type constructor will no longer throw when
```

Comment:
We could update the CHANGELOG.md to talk about the generic problem this solves rather than specifying a particular vendor specific environment. So something like:

"Use `AssemblyFileVersionAttribute` instead of `FileVersionInfo.GetVersionInfo` to get the SDK version attribute to ensure that it works when the assembly is not loaded directly from a file on disk."

Reply:
+1

Reply:
Fixed, thanks